### PR TITLE
Set web page title

### PR DESCRIPTION
**Pull request by QuentinBisson**
Set the web page title to be able to know to which prometheus I am talking to

Closes #https://github.com/giantswarm/giantswarm/issues/13368

## Checklist

I have:

- [x] Described why this change is being introduced
- [x] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

**Comment by QuentinBisson** [comment]
image

File: service/controller/resource/prometheus/resource.go [hunk 1]
```diff
@@ -153,7 +153,7 @@ func toPrometheus(v interface{}, config Config) (metav1.Object, error) {
 	}
 
 	image := fmt.Sprintf("%s/giantswarm/prometheus:%s", config.Registry, config.PrometheusVersion)
-
+	pageTitle := fmt.Sprintf("%s's Prometheus", key.ClusterID(cluster))
```
Comment (JosephSalisbury) [inline service/controller/resource/prometheus/resource.go:156]:
```suggestion
	pageTitle := fmt.Sprintf("%s Prometheus", key.ClusterID(cluster))
```

the possessive feels strange, but no strong opinion

File: service/controller/resource/prometheus/resource.go [hunk 2]
```diff
@@ -171,7 +171,11 @@ func toPrometheus(v interface{}, config Config) (metav1.Object, error) {
 			PodMetadata: &promv1.EmbeddedObjectMetadata{
 				Labels: labels,
 			},
-			Image:    &image,
+
```
Comment (JosephSalisbury) [inline service/controller/resource/prometheus/resource.go:174]:
```suggestion
```
is this newline good? it looks like we don't have them elsewhere in the struct

File: service/controller/resource/prometheus/resource.go [hunk 3]
```diff
@@ -153,7 +153,7 @@ func toPrometheus(v interface{}, config Config) (metav1.Object, error) {
 	}
 
 	image := fmt.Sprintf("%s/giantswarm/prometheus:%s", config.Registry, config.PrometheusVersion)
-
+	pageTitle := fmt.Sprintf("%s Prometheus", key.ClusterID(cluster))
```
Comment (mig4) [inline service/controller/resource/prometheus/resource.go:156]: I'd vote for including the installation name and no `'s` (like in tests), so e.g. `gauss/s06tw Prometheus`.

**Comment by QuentinBisson** [comment]
image
cc @mig4

**Review by JosephSalisbury — APPROVED** [review]
niceee

**Pull request merged**